.eslintrc

```diff
@@ -77,12 +101,35 @@
                         "disallowTypeAnnotations": true
                     }
                 ],
-                "@typescript-eslint/no-use-before-define": ["error", "nofunc"]
+                "@typescript-eslint/no-use-before-define": [
+                    "error",
+                    "nofunc"
+                ]
+            }
+        },
+        {
+            "files": [
+                "**/test/**/*.js",
+                "**/test/**/*.ts",
+                "**/test/**/*.tsx"
+            ],
+            "rules": {
+                "no-console": "off",
+                "jsdoc/require-returns-description": "off",
+                "jsdoc/require-param-description": "off",
+                "max-nested-callbacks": [
+                    "error",
+                    5
+                ],
+                "@typescript-eslint/no-non-null-assertion" : "off"
```

Comment:
This is the new overrides to disable some rules on the test code.

Reply:
Do we have a lot of non-null assertions in test code?

Reply:
Some, but probably don't need to disable the rule.  
I removed the rule override and adjusted the code where appropriate. 
https://github.com/SAP/open-ux-tools/pull/689/commits/e33874490b876aa1139fcb5780fcfb7dd624156e